setup.py

```diff
@@ -19,7 +19,7 @@
     packages={"galaxykit"},
     url="https://github.com/hendersonreed/galaxykit/",
     version="0.4.0",
-    install_requires=["requests", "simplejson"],
+    install_requires=["requests", "simplejson", "ansible", "orionutils"],
```

Comment:
I'm not keen on adding all of ansible core as a dependency. I think galaxykit should stay untethered.

Reply:
Now that the upload_artifact code as been ported, nothing else uses ansible and I've removed it from the requirements.